validphys2/src/validphys/photon/structure_functions.py

```diff
@@ -0,0 +1,80 @@
+import pineappl
+import numpy as np
+from scipy.interpolate import RectBivariateSpline
+
+class StructureFunction :
+    def __init__(self, path_to_fktable, pdfs):
```

Comment:
It would be good to have some documentation as to what this does and what are its arguments.